.github/CONTRIBUTING.md

```diff
@@ -20,13 +20,14 @@ This will create a new directory on your machine called "GoVote", that you shoul
 $ cd GoVote
 ```
 
-The project has been properly cloned at this point, and it is time to get it up and running. Run the following commands to install the project's dependencies, and start the project's development server:
+The project has been properly cloned at this point, and it is time to get it up and running. Run the following commands to build the govote image, create and seed the database, and start the containers.
 ```
-$ npm install
-$ npm run start
+$ docker-compose build # Builds the govote image
```

Comment:
Could we alias these as npm scripts as well in our package.json? I.e.

```
"scripts": {
  "docker:build": "docker-compose build",
  "docker:createTables": "docker-compose exec govote node ./bin/create-tables.js",
  "docker:seed": "docker-compose exec govote node ./bin/etl.js",
  "docker:dev": "docker-compose up -d"
}
```

Or something along those lines. We could then chain them to have a full:
```
"docker:rebuildDB": "npm run docker:build && npm run docker:seed"
```

Reply:
That's a great idea!